section2/exercises/if_statements.rb

```diff
@@ -61,5 +64,12 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
+cups_of_flour = 3
 has_sauce = true
+
+if
+  cups_of_flour >=2 && has_sauce == true
+  puts "I can make a pizza"
+else cups_of_flour < 2 || has_sauce != true
```

Comment:
Because this is an "else", it is a catch-all statement so you don't need to declare the condition :) Declaring the conditions would occur with "elsif", but you else is going to run if none of the previous conditions were met.